app/assets/javascripts/components/messages_view.coffee

```diff
@@ -0,0 +1,19 @@
+@MessagesView = React.createClass
+  displayName: 'MessagesView'
+
+  render: ->
+    {ul, li, div} = React.DOM
+    any = @props.messages.length > 0
+
+    if any
```

Comment:
`return null if @props.messages.length > 0`

tijdelijke variabele is niet leesbaarder dan een simpele length check.